### ISSUE TITLE
Database error when running campaign:update

### ISSUE THREAD

**Issue by zapur1** [mautic/mautic#685]
Hi,
I seem to have broken one of our campaigns.

The error is as follows and occurs when the campaign:update cron job is run
```
 [Doctrine\DBAL\DBALException]                                                                                                                                                                                                                                                                                                                                                              
  An exception occurred while executing 'SELECT max(ll.lead_id) as max_id, count(distinct(ll.lead_id)) as lead_count FROM lead_lists_leads ll WHERE (ll.leadlist_id IN ()) AND (ll.manually_removed = ?) AND (ll.date_added <= '2015-07-19 06:42:58') AND (ll.lead_id NOT IN (SELECT cl.lead_id FROM campaign_leads cl WHERE cl.campaign_id = 9)) ORDER BY ll.lead_id ASC' with params [0]:  
  SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')) AND (ll.manually_removed = 0) AND (ll.date_added <= '2015-07-19 06:42:58') AN' at line 1                                                                                                                                                                                                               [PDOException]                                                                                                                                                                                                                                                                              
  SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ')) AND (ll.manually_removed = 0) AND (ll.date_added <= '2015-07-19 06:42:58') AN' at line 1  
```
This occurred after creating a lead by submitting a form that is attached to the campaign. The lead was then deleted and created again by submitting the form. In the middle of this I was also editing the form.

I have tried clearing the file cache, is there a database cache that might be the source of the error??

Thanks,
Zac

**Comment by zapur1** [comment]
It appears that time and running the cron jobs multiple times has resolved the issue, now the campaign:update cron job appears to complete but has the following
```
 [Doctrine\ORM\ORMInvalidArgumentException]                                                                                                                                                                                                                                                                                                                                                                                         A new entity was found through the relationship 'Mautic\CampaignBundle\Entity\LeadEventLog#campaign' that was not configured to cascade persist operations for entity: Mautic\CampaignBundle\Entity\Campaign with ID #8. To solve this issue: Either explicitly call EntityManager#persist() on this unknown entity or configure cascade persist  this association in the mapping for example @ManyToOne(..,cascade={"persist"}).  
```
I was able to resolve it by, linking the lead list/campaign form lead source block to the first action of the campaign. Is this how it would normally be setup? 
<img width="167" alt="mautic" src="https://cloud.githubusercontent.com/assets/1893622/8764952/3134ed40-2e39-11e5-8acb-d66e659a1c5f.png">

Thanks,
Zac

**Comment by alanhartless** [comment]
I believe this is related to #533 and is fixed in the just released 1.1.3.
